R/dataframe.R

```diff
@@ -200,6 +200,34 @@ add_rownames <- function(df, var = "rowname") {
   as_data_frame(cbind(rn, df))
 }
 
+#' Convert a column of tbl into rownames of data.frame.
+#'
+#' Inverted operation of add_rownames
+#'
+#' @param df Input data frame with rownames.
+#' @param var Name of variable to use
+#' @export
+#' @seealso \code{\link{add_rownames}}
+#' @examples
+#' tbl_df(mtcars)
+#'
+#' mtcars_tbl <- add_rownames(mtcars)
+#' mtcars_data.frame <- use_as_rownames(mtcars_tbl)
+use_as_rownames <- function(df, var = "rowname") {
+  stopifnot(is.data.frame(df))
+
+  if ( !var %in% colnames(df) ) {
+    stop(paste("No", var, "column in the colnames.") )
+  }
+
+  df <- as.data.frame(df)
```

Comment:
At least to me, `use_as_rownames` is to interact with the `base::data.frame` and `tbl` is trying to avoid `rownames`.  So I convert it into data.frame directly. Comments?